client/app/components/sidebar-menu/sidebar-menu.config.js

```diff
@@ -250,12 +250,18 @@ angular.module("App").run(($q, SidebarMenu, Products, User, constants, translato
             loadOnState: "app.microsoft.exchange"
         }, microsoftItem);
 
+        const exchangeProductTypes = {
```

Comment:
I've kept the "default state" feature, so if a new offer arrives, then the `Hosted` state is selected by default